messaging.fml.yaml

```diff
@@ -152,13 +155,13 @@ features:
               priority: 100
               max-display-count: 1
           notification-config:
-            polling-interval: 15 # minutes
+            refresh-interval: 120 # minutes (2 hours)
           messages:
-            test-notification:
-              title: "Test title"
-              text: "Test text"
+            re-engagement-notification:
+              title: notification_re_engagement_title
+              text: notification_re_engagement_text
```

Comment:
this referenced string takes a string arg - unsure whether this can still be used?